zad6/db/migrate/20181212114502_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :products do |t|
+      t.string :name
+      t.decimal :price, scale: 2, precision: 10
+      t.references :customer, foreign_key: true
```

Comment:
Relacja między produktami i customerami wg treści zadania:
```
* klient może zamówić wiele produktów
* produkt może być zamówiony tylko przez jednego klienta
```
A więc jest to relacja `1-*`. W tym przypadku do jednego customera może być przypisanych wiele produktów, a więc w tabeli `products` powinna się znaleźć kolumna zawierająca `customer_id`.